src/components/tree/_tree.scss

```
@@ -49,6 +59,20 @@
       margin: 0 5px 0 0;
       top: 2px;
       width: 15px;
+
+      &.rotate180-forward,
+      &.rotate180-backward {
+        animation-duration: 0.7s;
```

```suggestion
        animation-duration: 0.3s;
```
makes it end at the same time as the expand/collapse, what do you think?

---

Nice then maybe you dont need to worry about the problem i saw with the children animating